tests/acceptance.f2Qkb1

```diff
@@ -0,0 +1,911 @@
+Attaching to acceptance-tests_acceptance_1, acceptance-tests_mender-deployments_1, acceptance-tests_mender-api-gateway_1, acceptance-tests_mender-device-auth_1, acceptance-tests_mender-useradm_1, acceptance-tests_mender-inventory_1, acceptance-tests_mender-workflows-server_1, acceptance-tests_mender-gui_1, acceptance-tests_mender-mongo_1, acceptance-tests_minio_1
```

Comment:
I thought we had gitignore rules for these files...

Could you please remove these output files as well as the `tests/*_api.yml` and `tests/run-test-environment` files

Reply:
Oh, sorry, bad mistake. I force-pushed to remove them.